src/screens/IndexScreen.js

```diff
@@ -268,6 +270,13 @@ export const IndexScreen = ({ navigation, route }) => {
                 ListHeaderComponent={
                   showFilter ? (
                     <DropdownHeader {...{ query, queryVariables, data, updateListData }} />
+                  ) : subCategories?.length ? (
+                    <CategoryList
+                      navigation={navigation}
+                      data={subCategories}
+                      horizontal={false}
+                      noSectionHeader={true}
```

Comment:
in most cases we want a section header. i think it is easier to think and to explain with a prop `hasSectionHeader` here, that default to `true` and is disabled with `false` here in this on place.

Reply:
I edited the code. It's currently going through two different filters and showing whichever is needed.